### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pwltr**
This contents of this readme should be directed towards users looking to get started with Citadel.

- [x] Add Installation section
- [x] Add links to communitaction channels (Telegram, Matrix, Discord)
- [x] Add other relevant links (blog, wiki etc.)
- [x] Add Umbrel comparison (?)
- [x] Add Contributing section
- [x] Add License section
- [ ] Add Donation section
- [x] Update Roadmap
- [x] Update Repository table
- [x] Update description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,27 @@ Our node provides the latest apps, together with our own, up-to-date user interf
...
+
+### Debian or Ubuntu on x86 hardware
+
+[runcitadel.space/x86](https://runcitadel.space/x86)
+
+### Docker + Sysbox (experimental)
+
+[github.com/runcitadel/citadel-dev](https://github.com/runcitadel/citadel-dev)
+
+## 🧭 Roadmap
```
Comment (AaronDewes) [inline README.md:50]: You could replace the part below this with a link to the "Citadel development" GH project. This plan is outdated anyway (From 2021)

**Comment by pwltr** [comment]
What's the best way to accept lightning donations in the readme?

**Comment by AaronDewes** [comment]
> What's the best way to accept lightning donations in the readme?

Probably add a link to a donation page (Which I am setting up right now).

**Comment by pwltr** [comment]
@AaronDewes @WilliamConnatser
Please review: https://github.com/runcitadel/citadel/blob/patch-1/README.md

**Review by WilliamConnatser — APPROVED** [review]

**Pull request merged**